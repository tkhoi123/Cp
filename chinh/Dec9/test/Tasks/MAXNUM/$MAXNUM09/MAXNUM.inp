10
817653 697874 131821 141726 2120 924288 212042 2477 588316 283397 