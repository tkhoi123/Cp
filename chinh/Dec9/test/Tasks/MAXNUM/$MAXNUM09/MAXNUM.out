92428881765369787458831628339724772120422120141726131821